_redirects

```diff
@@ -179,6 +179,9 @@
 /docs/getting-started-guides/vagrant/     /docs/getting-started-guides/alternatives/ 301
 /docs/getting-started-guides/windows/While/     /docs/getting-started-guides/windows/ 301
 
+/docs/getting-started-guides/centos/* /docs/setup/independent/create-cluster-kubeadm/ 301
+/cn/docs/getting-started-guides/centos/* /docs/setup/independent/create-cluster-kubeadm/ 301
```

Comment:
cn docs are not actively maintained in this repo, so we don't need this line I think. cc @markthink @hanjiayao for confirmation.

Reply:
I've remove it.